_overviews/scala3-migration/plugin-kind-projector.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Kind Projector Migration
+type: section
+description: This section shows how to migrate from the kind-projector plugin to Scala 3 kind-projector syntax
+num: 25
+previous-page: plugin-intro
+next-page:
+---
+
+In the future, Scala 3 will use the `_` underscore symbol for placeholders in type lambdas---just as the underscore is currently used for placeholders in (ordinary) term-level lambdas.
+
+```scala
+
+```
+
+The new type lambda syntax is not enabled by default, to enable it, use a compiler flag `-Ykind-projector:underscores`. Note that enabling underscore type lambdas will disable usage of `_` as a wildcard, you will only be able to write wildcards using the `?` symbol. 
+
+If you wish to cross-compile a project for Scala 2 & Scala 3 while using underscore type lambdas for both, you may do so starting with [kind-projector](https://github.com/typelevel/kind-projector) version `0.13.0` and up and Scala 2 versions `2.13.6` and `2.12.14`.
+To enable it, add the compiler flags `-Xsource:3 -P:kind-projector:underscore-placeholders` to your build.
+As in Scala 3, this will disable usage of `_` as a wildcard, however, the flag `-Xsource:3` will allow you to replace it with the `?` symbol. 
+
+The following `sbt` configuration will set up the correct flags to cross-compile with new syntax:
+
+```scala
+ThisBuild / scalacOptions ++= {
+  CrossVersion.partialVersion(scalaVersion.value) match {
+    case Some((3, _)) => Seq("-Ykind-projector:underscores")
+    case Some((2, 13)) => Seq("-Xsource:3", "-P:kind-projector:underscore-placeholders"))
```

Comment:
@adpi2 The options work in `2.12` as well, although I’m not sure if these guides cover 2.12.